renderdoc/driver/ihv/arm/official/lizard/README.md

```diff
@@ -0,0 +1,38 @@
+## Lizard
+
+Lizard is a library to capture ARM hardware counters on Android devices.
+
+## How to use with RenderDoc
+
+1. Start RenderDoc on desktop and connect to the device. During this, the RenderDoc APKs will be installed
+   if it's not already on the device and the library will be available if ARM Mali GPU is detected.
+
+2. Capture an application on the device.
+
+3. Start the replay of the capture.
+
+4. To view the counters in the RenderDocs' Window menu, there is the "Performance Counter Viewer" menu item.
+
+5. In the "Performance Counter Viewer" window the "Capture counters" button will provide a list of available counters.
+   There should be a group for ARM counters. Selecting one or more counters and clicking "Sample counters" will
+   gather the values for those counters.
```

Comment:
This README seems misplaced or stale. A third party library shouldn't have instructions for RenderDoc's UI like this.